pkgs/development/python-modules/pyelftools/default.nix

```diff
@@ -20,7 +20,7 @@ buildPythonPackage rec {
     hash = "sha256-+T5C0ah2oj5E8fWaQbuzYRVgD5bSiUbaArrlxNLojvw=";
   };
 
-  doCheck = stdenv.hostPlatform.system == "x86_64-linux";
+  doCheck = stdenv.hostPlatform.system == "x86_64-linux" && !stdenv.hostPlatform.isMusl;
```

Comment:
This change could also do with an explanation in the commit message, or in a comment.  What goes wrong?  What would have to change before we could re-enable the tests?

Reply:
I didn't look closely at the errors, as I just wanted to get it working. So I don't know what would have to change before the tests could be re-enabled.

Reply:
`@@ aborting - 'test/external_tools/readelf -e' returned '127'`

Reply:
So the 'source' contains a readelf binary built against glibc. I will try to replace it with our own readelf so that we can leave the tests enabled.

Reply:
Nope, using another readelf binary makes a lot of tests fail (also on glibc). Adding a comment to the commit message but leaving tests disabled.

Reply:
Sounds good.